Components/Bites/src/OgreApplicationContextBase.cpp

```diff
@@ -28,6 +28,13 @@ ApplicationContextBase::ApplicationContextBase(const Ogre::String& appName)
 {
     mAppName = appName;
     mFSLayer = new Ogre::FileSystemLayer(mAppName);
+
+    if (char* val = getenv("OGRE_CONFIG_DIR"))
+    {
+        Ogre::String configDir = Ogre::StringUtil::standardisePath(val);
+        mFSLayer->setConfigPaths({ configDir });
```

Comment:
```suggestion
        mFSLayer->setConfigPaths({ configDir });
        LogManager::getSingleton().logMessage(
                "setting ConfigFile Path from OGRE_CONFIG_DIR environment variable");
```

Reply:
doesn't work, because the singleton is still NULL here.